include/CL/opencl.hpp

```diff
@@ -4618,6 +4645,41 @@ class Image : public Memory
         }
         return param;
     }
+
+#if defined(cl_ext_image_requirements_info)
+    static cl_int getImageRequirementsInfoExt(const Context &context,
+        cl_mem_flags flags,
+        ImgReqInfoExt param_name,
+        size_type param_value_size,
+        void* param_value,
+        size_type* param_value_size_ret,
+        const cl_image_format* image_format = nullptr,
+        const cl_image_desc* image_desc = nullptr,
+        const cl_mem_properties* properties = nullptr)
```

Comment:
I'm not going to lose much sleep over this but it's a little odd to have these parameters with default values in this specific order, and I wonder if it makes more sense to have specific overloads instead.